packages/components/pie-webc-core/src/test/mixins/formControlMixin/formControlMixin.browser.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { test, expect } from '@sand4rt/experimental-ct-web';
+import { MockComponent } from './MockComponent.ts';
+
+test.describe('FormControlMixin', () => {
+    // IMPORTANT: Mounting and Unmounting the component before each test ensures that any tests that do not explicitly
+    // mount the component will still have it available in Playwright's cache (loaded and registered in the test browser)
+    test.beforeEach(async ({ mount }) => {
+        await (await mount(MockComponent)).unmount();
+    });
+
+    test.describe('form property', () => {
+        test('should not have an associated form when no form exists', async ({ page, mount }) => {
+            // Arrange
+            await mount(MockComponent);
+
+            const isFormAssociated = await page.evaluate(() => {
+                const component = document.querySelector('form-control-mixin-mock');
+                const form = component?.form;
+
+                return !!form;
+            });
+
+            // Assert
+            expect(isFormAssociated).toBe(false);
+        });
+
+        test('should return the associated form when inside of a form', async ({ page }) => {
+            // Arrange
+            await page.evaluate(() => {
+                const formHTML = `
+                <form id="testForm" action="/foo" method="POST">
+                    <input type="text" id="username" name="username" required>
+                    <input type="password" id="password" name="password" required>
+                    <form-control-mixin-mock></form-control-mixin-mock>
+                </form>
+                `;
+                document.body.innerHTML = formHTML;
```

Comment:
You could probably make use of `await page.setContent(formHTML)`

Reply:
excellent idea! Will update